crypto/x509/x509_lcl.h

```diff
@@ -16,7 +16,7 @@
 struct X509_VERIFY_PARAM_st {
     char *name;
     time_t check_time;          /* Time to use */
-    unsigned long inh_flags;    /* Inheritance flags */
+    uint32_t inh_flags;         /* Inheritance flags */
```

Comment:
Suppose LONG is 64 bits.  Then the structure type will not match the API/ABI parameter type.  And binaries will not port across.